litmus/PTX/SB-strong.litmus

```diff
@@ -1,4 +1,5 @@
-PTX Co-Total-strong
+PTX SB-strong
```

Comment:
I don't think this is a SB pattern. The SB has a store followed by a load in each thread

Reply:
Updated